keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -0,0 +1,1261 @@
+- [Feature gate][feature gate] lifecycle
+- [Deprecation policy][deprecation-policy]
+
+Clearly define what graduation means by either linking to the [API doc
+definition](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning)
+or by redefining what graduation means.
+
+In general we try to use the same stages (alpha, beta, GA), regardless of how the
+functionality is accessed.
+
+[feature gate]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[maturity-levels]: https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+Below are some examples to consider, in addition to the aforementioned [maturity levels][maturity-levels].
+-->
+#### Alpha to Beta
+
+- Feature implemented behind a feature flag
+- Initial e2e tests completed and enabled
+- Integrate API feedback from users and community
+-	Proven cross-components consistency (ideally via tests)
+-	Handling of topology manager and memory manager use-cases
+-	Finish Identified Code refactoring of common building blocks (look at common pieces in all plugin-frameworks  in kubelet) 
+-	Look to what makes sense to leave inside Kubelet due to latency and use case requirements. Introduce community CCI drivers repo
+-	Similar to Kubernetes Scheduler Plugin repo
+-	Have plugins specific to common use cases or environments
+-	Smooth integration with scheduler extensions/ plugins …
+
+#### Beta to GA
+
+- Gather feedback from developers and surveys and integrate it
+-	Successful adoption across users
+-	Proven Components Correctness
+-	Performance tests completion
+- N examples of real-world usage
+
+#### Deprecation
+
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to maintain previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to make use of the enhancement?
+-->
+
+### Version Skew Strategy
+
+<!--
+If applicable, how will the component handle version skew with other
+components? What are the guarantees? Make sure this is in the test plan.
+
+Consider the following in developing a version skew strategy for this
+enhancement:
+- Does this enhancement involve coordinating behavior in the control plane and
+  in the kubelet? How does an n-2 kubelet without this feature available behave
+  when this feature is used?
+- Will any other components on the node change? For example, changes to CSI,
+  CRI or CNI may require updating that component before the kubelet.
+-->
```

Comment:
I think this deserves explanation, especially because we will need to handle the kubelet <=> cci drivers API and the upgrade/downgrade strategy of CCI drivers

Reply:
Added.  Please review.